### ISSUE TITLE
Dynamic Group Anchoring bugged since Cata (always using the latest WA Alpha version)

### ISSUE THREAD

**Issue by Fileor** [WeakAuras/WeakAuras2#5060]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues.

### Description

**since cata bugged** (worked fine in wotlk via "Screen/Parent Group"):
![3_screenParetnGrp_workedFineInWotlk](https://github.com/WeakAuras/WeakAuras2/assets/123123388/0e8e366a-f92b-4969-9a31-aecdbfdf476f)

**how it should be saved** (correct position/anchor below target nameplate – toggle view "greenEye" shows it once but not saving it like it did in wotlk):
![2_thisPositionShouldBeSaved_eitherViaScreenParentGGroup_OR_viaSelectFrame](https://github.com/WeakAuras/WeakAuras2/assets/123123388/2764e6b6-9a1e-4325-b96d-d365b5df4947)

not saving it if switching between "Select Frame" & "Screen/Parent Group":
![1_NotSavingThePosition(Anchor)](https://github.com/WeakAuras/WeakAuras2/assets/123123388/dcab05a3-55c4-485d-97d6-64c9fa4388d3)

the same anchoring (in this case "WeakAuras:NameplateAnchorTarget") **still works fine for normal/NON dynamic groups**/Auras...

### WeakAuras Version

5.12.9-8 (latest Alpha)

### World of Warcraft Flavor

Cataclysm Classic

### World of Warcraft Region

EU

### Tested with only WeakAuras

I got this issue with only WeakAuras enabled

### Reproduction Steps

see screenshots & snip from "WeakAuras.lua"

### Export String

```Text
example:

		["BigIcon@tarNP_LC&majorBuffs"] = {
			["grow"] = "HORIZONTAL",
			["controlledChildren"] = {
				"PlayerLcTarNp", -- [1]
				"majorBuffs/debuffs@PlayerTarNp", -- [2]
			},
			["borderBackdrop"] = "Blizzard Tooltip",
			["parent"] = "TargetNP",
			["yOffset"] = -35,
			["sortHybridTable"] = {
				["PlayerLcTarNp"] = true,
			},
			["borderColor"] = {
				0, -- [1]
				0, -- [2]
				0, -- [3]
				1, -- [4]
			},
			["space"] = 0,
			["actions"] = {
				["start"] = {
				},
				["finish"] = {
				},
				["init"] = {
				},
			},
			["triggers"] = {
				{
					["trigger"] = {
						["debuffType"] = "HELPFUL",
						["type"] = "aura2",
						["spellIds"] = {
						},
						["subeventSuffix"] = "_CAST_START",
						["unit"] = "player",
						["subeventPrefix"] = "SPELL",
						["event"] = "Health",
						["names"] = {
						},
					},
					["untrigger"] = {
					},
				}, -- [1]
			},
			["columnSpace"] = 1,
			["radius"] = 200,
			["useLimit"] = true,
			["align"] = "CENTER",
			["selfPoint"] = "CENTER",
			["alpha"] = 1,
			["anchorFrameFrame"] = "WeakAuras:NameplateAnchorTarget",
			["rotation"] = 0,
			["arcLength"] = 360,
			["authorOptions"] = {
			},
			["subRegions"] = {
			},
			["gridType"] = "RD",
			["fullCircle"] = true,
			["load"] = {
				["talent"] = {
					["multi"] = {
					},
				},
				["spec"] = {
					["multi"] = {
					},
				},
				["class"] = {
					["multi"] = {
					},
				},
				["size"] = {
					["multi"] = {
					},
				},
			},
			["rowSpace"] = 1,
			["backdropColor"] = {
				1, -- [1]
				1, -- [2]
				1, -- [3]
				0.5, -- [4]
			},
			["internalVersion"] = 73,
			["animate"] = false,
			["animation"] = {
				["start"] = {
					["easeStrength"] = 3,
					["type"] = "none",
					["duration_type"] = "seconds",
					["easeType"] = "none",
				},
				["main"] = {
					["easeStrength"] = 3,
					["type"] = "none",
					["duration_type"] = "seconds",
					["easeType"] = "none",
				},
				["finish"] = {
					["easeStrength"] = 3,
					["type"] = "none",
					["duration_type"] = "seconds",
					["easeType"] = "none",
				},
			},
			["scale"] = 1,
			["centerType"] = "LR",
			["border"] = false,
			["borderEdge"] = "Square Full White",
			["regionType"] = "dynamicgroup",
			["hybridPosition"] = "hybridFirst",
			["sort"] = "hybrid",
			["hybridSortMode"] = "ascending",
			["xOffset"] = 0,
			["anchorFrameParent"] = false,
			["constantFactor"] = "RADIUS",
			["stagger"] = 0,
			["borderOffset"] = 4,
			["anchorPoint"] = "CENTER",
			["borderInset"] = 1,
			["id"] = "BigIcon@tarNP_LC&majorBuffs",
			["gridWidth"] = 5,
			["frameStrata"] = 6,
			["anchorFrameType"] = "SCREEN",
			["stepAngle"] = 15,
			["uid"] = "EQ2guJXbr)d",
			["borderSize"] = 2,
			["limit"] = 4,
			["conditions"] = {
			},
			["information"] = {
			},
			["config"] = {
			},
```

**Comment by emptyrivers** [comment]
I created a dynamic group, and anchored it to the screen, and everything seems to be in order. I then anchored that dynamic group to another WA, and it still seemed fine.

Attach an actual import string (*not* the debug table), or provide better steps to reproduce, please.
![image](https://github.com/WeakAuras/WeakAuras2/assets/24878501/a8de153b-10d5-4966-a5c4-791407e4db08)

**Comment by Stanzilla** [comment]
Try with and without this enabled and see if it fixes it?

![image](https://github.com/WeakAuras/WeakAuras2/assets/75278/facc51b0-e1bf-4665-9797-ee55c44d0f03)

**Comment by mrbuds** [comment]
This is fixed with WeakAuras 5.13.0
